auto_nag/scripts/spambug.py

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bugbug_utils import get_bug_ids_classification
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.utils import nice_round
+
+
+class SpamBug(BzCleaner):
+    def __init__(self):
+        super().__init__()
+
+    def description(self):
+        return "[Using ML] Detect spam bugs"
+
+    def columns(self):
+        return ["id", "summary", "confidence", "autofixed"]
+
+    def sort_columns(self):
+        return lambda p: (-p[2], -int(p[0]))
+
+    def get_bz_params(self, date):
+        start_date, end_date = self.get_dates(date)
+
+        return {
+            "include_fields": ["id", "groups", "summary"],
+            # Ignore closed bugs.
+            "bug_status": "__open__",
+        }
```

Comment:
We can reduce the number of bugs to retrieve by checking if the reporter doesn't have a mozilla address (as they are surely not spam).

@calixteman can you think of any other bugs to skip?

Reply:
> We can reduce the number of bugs to retrieve by checking if the reporter doesn't have a mozilla address (as they are surely not spam).

Adding `f1=reporter&v1=@mozilla&o1=notsubstring` should take care of it.

Reply:
probably we can filter after the query ran in using `people::People::is_mozilla`: some people may be mozilla but with a different bugmail.
Anyway, I don't see any other pattern we could filter out.

Reply:
OK, let's do both:
- if we just filter out after the query, we spend time classifying some bugs needlessly;
- if we just filter before the query, we will miss some people who are actually mozillians.